beacon-chain/blockchain/forkchoice/metrics.go

```diff
@@ -57,6 +57,10 @@ var (
 		Name: "beacon_current_validators",
 		Help: "Number of status=pending|active|exited|withdrawable validators in current epoch",
 	})
+	sigFailsToVerify = promauto.NewCounter(prometheus.CounterOpts{
+		Name: "att_signature_failed_to_verify",
+		Help: "Number of attestation signatures that failed to verify",
```

Comment:
Can you clarify the intention of this metric? What I see in the code is that you increment this when a signature failed to verify from the cache and then you manually recheck it and then it succeeded to verify

Reply:
It's manually recheck with the cache off

Clarified in comments